### DISCUSSION TITLE
How to integrate assimp with filament in Android ?

### DISCUSSION THREAD

**Discussion by udayk07** [google/filament#3802] [Q&A] [answered]
I am trying to load collada, obj, stl, fbx format 3d models ( models will be loaded dynamically) using filament engine. As filament supports gltf only, is there any workflow/ example projects to integrate with assimp to load above format models with filament ? or is there any option to load above models with filament ?

**Comment by prideout** [comment, marked as answer]
For an example you can look at `libs/filamentapp/src/MeshAssimp.cpp`.

**Reply by udayk07** [reply]
Sure. Is there any java bindings are there ( to use with sceneform, like gltfio ) ?

**Reply by prideout** [reply]
No, in fact MeshAssimp is not an officially supported library (we are a very small team) so we do not provide Java bindings for it.  I just thought it would be helpful example.
